Makefile

```diff
@@ -71,7 +71,7 @@ help:
 	@echo "test - run python tests with the default Python"
 	@echo "test-deps - install the test dependencies"
 	@echo "test-js-karma - run newer js tests in terminal; primarily for CI build"
-	@echo "test-mocha-karma - run older js tests that have transitioned to karma in the terminal"
+	@echo "test-js-old - run older js tests that have transitioned to karma in the terminal"
```

Comment:
We didn't choose this because the code is still modified and updated regularly so it's not 'old'. If you'd like the suites to be more consistent then `test-js-karma` should be renamed to `test-jasmine-karma`